modules/network-observability-operator-install.adoc

```diff
@@ -9,7 +9,9 @@ You can install the Network Observability Operator using the {product-title} web
 
 .Prerequisites
 
-* Installed Loki. It is recommended to install Loki using the link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.6].
+* Installed Loki. It is recommended to install Loki using the link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.7].
+* One of the following supported architectures is required: `amd64`, `ppc64le`, or `arm64`.
+* Red Hat Enterprise Linux (RHEL) 9
```

Comment:
I don't think we should say rhel9 is a pre-req, it's more complicated - for instance, a counter-example is that we support ocp4.12 which uses rhel8 on its nodes. The OS installed on nodes (like rhel8) can differ from the OS used in our containers (rhel9).
It would rather be something like:
`* Any CPU supported by Red Hat Enterprise Linux (RHEL) 9`